### ISSUE TITLE
Ublock Origin block EVERY Googlevideo stream

### ISSUE THREAD

**Issue by VORAPIS** [uBlockOrigin/uAssets#20384]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is NOT an issue with [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

This extension block EVERY request to Googlevideo
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/102152194/5b07dddb-c816-472b-b3f8-16419e29717c)

### A specific URL where the issue occurs.

https://www.youtube.com/watch?v=jZGpkLElSu8

### Steps to Reproduce

1. Load any youtube video

### Expected behavior

Youtube video load

### Actual behavior

Youtube video not load

### uBO version

 1.52.2

### Browser name and version

Cent Browser based on Chromium 102

### Operating System and version

Windows 7

**Comment by VORAPIS** [comment]
All video streams is blocked by Ublock, player cannot download anything
![image](https://github.com/uBlockOrigin/uAssets/assets/102152194/566f0ef9-f67e-4bb4-ada0-2104bedf3523)

**Comment by gorhill** [comment]
You will have to give us all the details, it seems you are telling everything. What am I looking at in that screenshot? That doesn't seem to be the regular Youtube video player.

**Comment by VORAPIS** [comment]
i am using a userscript that forces youtube to use a older version of youtube player, and as far as i can see, the issue is caused by the fact that it requests streams using XMLHttpRequest

**Comment by stephenhawk8054** [comment]
Hmmm... Normal googlevideo is `post` requests as far as I know. Can you test in another chromium browser with latest version?

**Comment by stephenhawk8054** [comment]
> if someone dont, player start using Xhr too and all requests are blocked!!

Yes, the filter is added due to youtube's anti-adblock flawed codes. Just discuss calmly please.

**Comment by VORAPIS** [comment]
the !! wasnt meant to indicate a certain feeling but its very important to acknowledge that youtube player js still have code for both type of request, so this issue can affect anyone randomly

**Comment by stephenhawk8054** [comment]
Yes, we know. Please let us time to investigate

Sigh

...

**Comment by stephenhawk8054** [comment]
YouTube is the one putting flawed codes in their anti-adblock scirpt, not us...

**Comment by gorhill** [comment]
> GET requests via fetch (not blocked by uBO; the filter is seemingly only for XHR)

Fetch requests are reported as xhr in the logger.
